core/src/main/scala/akka/persistence/cassandra/EventsByTagMigration.scala

```diff
@@ -186,7 +186,7 @@ class EventsByTagMigration(system: ActorSystem)
           val startingSeqFut = tagScanningStartingSequenceNr(pid)
           for {
             tp <- lookupTagProgress(pid)
-            _ <- sendTagProgress(pid, tp, tagWriters)
+            _ <- persistenceIdStarting(pid, tp, tagWriters, system.deadLetters)
```

Comment:
`deadLetters`??

Reply:
This is a offline migration script so no watching required